packages/wp-now/src/tests/wp-now.spec.ts

```diff
@@ -514,6 +514,42 @@ describe('Test starting different modes', () => {
 		expect(themeName.text).toContain('Twenty Twenty-Three');
 	});
 
+	/**
+	 * Test that startWPNow compresses the text files.
+	 */
+	test('startWPNow compresses the text files', async () => {
+		const projectPath = path.join(tmpExampleDirectory, 'theme-with-assets');
+		const config = await getWpNowConfig({ path: projectPath });
+		const { options } = await startWPNow(config);
+
+		const html = await fetch(options.absoluteUrl);
+		expect(html.headers.get('content-encoding')).toBe('gzip');
+
+		const style = await fetch(
+			`${options.absoluteUrl}/wp-content/themes/theme-with-assets/style.css`
+		);
+		expect(style.headers.get('content-encoding')).toBe('gzip');
+
+		const javascript = await fetch(
+			`${options.absoluteUrl}/wp-content/themes/theme-with-assets/javascript.js`
+		);
+		expect(javascript.headers.get('content-encoding')).toBe('gzip');
```

Comment:
The only assertion that is working is for Style.

Here is the header for html in case it helps:

```
HeadersList {
  [Symbol(headers map)]: Map(5) {
    'x-powered-by' => 'Express',
    'date' => 'Fri, 02 Jun 2023 14:57:01 GMT',
    'connection' => 'keep-alive',
    'keep-alive' => 'timeout=5',
    'transfer-encoding' => 'chunked'
  },
  [Symbol(headers map sorted)]: null
}
```